modules/veteran_verification/METADATA.yml

```diff
@@ -0,0 +1,96 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: Veteran Verification
+  description: |
+
+    ## Background
+
+    The use-case of this API is to allow third-parties to do the following after
+    receiving authorization using an Open ID Connect flow: 
+
+    #### Veteran Status - Request confirmation from the VA of an individual's Veteran status 
+
+    The Veteran Verification API passes requests through to eMIS, the Enterprise
```

Comment:
Veteran Verification _is_ an API, but in this case I think we mean the "Veteran Confirmation" endpoint.

Reply:
Maybe "Veteran Status endpoint"? I feel like calling it Confirmation when we are adding another confirmation endpoint will just be confusing